### ISSUE TITLE
"Cannot find name abstract"

### ISSUE THREAD

**Issue by SomeKittens** [Microsoft/TypeScript-Sublime-Plugin#360]
```typescript
abstract class Node {
  constructor() {}
}
```

Get the error on above code.  `tsc` in the terminal does not complain, only in Sublime.  Running `v0.1.8`

**Comment by SomeKittens** [comment]
Restarted Sublime again and all this was fixed.  Weird, some race condition or somesuch.  Works now!
